frontend/web/js/api.ts

```diff
@@ -228,19 +228,23 @@ export class ApiService {
 export async function pollJob<T extends ApiJobStatus>(
   checkJobFn: () => Promise<T>,
   waitTime: number,
-  maxRetries: number = 10, // Default maximum retries
+  maxRetries: number = 20 // allow more retries for long jobs
```

Comment:
Do we want to limit the number of retries or should we have a timer instead? What do you think is most intuitive?

Say that we would like to increase the timeout to accommodate a job that can run for 3 minutes?

Reply:
I think number of retries is the least amount of code. Setting a max time means you're still going to have to set time inbetween retries. Or?